finat/ufl/finiteelementbase.py

```diff
@@ -143,7 +143,7 @@ def symmetry(self):  # FIXME: different approach
 
     def _check_component(self, domain, i):
         """Check that component index i is valid."""
-        sh = self.value_shape(domain.geometric_dimension())
```

Comment:
Should this function still depend on the domain, or could it just be renamed to `_check_reference_component`?